modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc

```diff
@@ -39,8 +39,6 @@ You will receive an email sent to the address you entered.
 This completes the registration and associates the email with a Redpanda account. 
 
 . On the **Accept your invitation to sign up** page, click **Sign up** or **Log in**. 
-+
-You can now create namespaces and clusters in your organization.
```

Comment:
Why are we removing this line?

Reply:
I can put it back replacing resource groups!